src/core/oned/MultiFormatOneDReader.ts

```diff
@@ -26,6 +26,7 @@ import BitArray from './../common/BitArray';
 import ITFReader from './ITFReader';
 import MultiFormatUPCEANReader from './MultiFormatUPCEANReader';
 import NotFoundException from '../NotFoundException';
+import Code39Reader from './Code39Reader';
```

Comment:
Let's order this alphabetically.

Reply:
These imports aren't ordered at all. And there are two groups:

https://github.com/jojastahl/library/blob/2b7f0470699aa5593880775ee8f62ab1fed6525c/src/core/oned/MultiFormatOneDReader.ts#L19-L29

Shall I put the two groups together and order all alphabetically? Or in which group would you like to see Code39Reader?

Reply:
I took a look at the ZXing original code and I can confirm what you're saying. Let's make our ordering for now by putting the imports in alphabetic order. 🙂